util/portmuxusr.c

```diff
@@ -58,7 +58,6 @@ int gbl_pmux_route_enabled = 1;
 #define PORTMUX_SET_NO_LINGER() 1
 #define PORTMUX_USE_ECHO_GET() 1
 #define PORTMUX_USE_REFACTORED_GET() 1
-#define PORTMUX_VALIDATE_NAMES() 1
```

Comment:
Keep the macro. Just change it to 0.

Reply:
Dont have a clue why i removed this. Putting back.